packages/polaris-viz/src/components/FunnelChartNext/Chart.tsx

```diff
@@ -46,22 +42,22 @@ import {
 export interface ChartProps {
   data: DataSeries[];
   tooltipLabels: FunnelChartNextProps['tooltipLabels'];
-  xAxisOptions: Required<XAxisOptions>;
-  yAxisOptions: Required<YAxisOptions>;
-  renderScaleIconTooltipContent?: () => ReactNode;
+  seriesNameFormatter: LabelFormatter;
+  labelFormatter: LabelFormatter;
   percentageFormatter?: (value: number) => string;
+  renderScaleIconTooltipContent?: () => ReactNode;
```

Comment:
This prop is needed to render the content within the tooltip, because the scaling feature is dependant on the type of data passed into the chart (we only trigger scaling when the smallest segment is < 10% that of the largest), the tooltip may or may not be rendered, thats why I decided to leave the prop as optional. However for our current use cases, this prop is necessary